.lando.yml

```diff
@@ -56,11 +60,6 @@ services:
         - seccomp:unconfined
       command: ["/usr/local/bin/supervisord", "-c", "/etc/supervisord.conf"]
 
-events:
-  post-start:
-    - node: /bin/sh -c "chmod -R 775 /var/www"
```

Comment:
We need this for the Windows machines. Otherwise, their environments won't correctly build the theme.

Reply:
I put this back, including the theme build. Both make sense to be there, shouldn't have been removed.